src/Components/ServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Extensions;
+using System.Collections.Generic;
+
+namespace Components
+{
+	public static class ServiceExtensions
+	{
+		public static IServiceCollection UseComponentSystem(this IServiceCollection services)
+		{
+			services.TryAdd(DefaultServices());
```

Comment:
The issue I have primarily is that it's public which [Haok mentioned](https://github.com/aspnet/Mvc/issues/1634#issuecomment-65720794). Also by privatizing it you are promoting reuse which totally doesn't make sense here.